Configuring Tailwind and Output Component

HRC-123 opened the pull request:
Issues Resolved :  #3 #6 
@ravinder-chadha  Hello Sir,
I have added Tailwind configuration and  Output Component

> The Output Component looks like this:
- Initial
![image](https://github.com/iRootsOrg/CodeBase/assets/106550364/2334dc6a-78d9-4f88-b60c-cd52f1c9129e)

- When there is an error:
![image](https://github.com/iRootsOrg/CodeBase/assets/106550364/60ca6352-0a05-4e0f-b0c2-6abf3700abb7)

- When there is an output:
![image](https://github.com/iRootsOrg/CodeBase/assets/106550364/14f219ef-3ba3-4156-af57-2a7a9b00b616)

- Different types in output like graphs etc.
![image](https://github.com/iRootsOrg/CodeBase/assets/106550364/80ebe1f6-4615-489c-8db0-97329cfb0d31)


As I did not have any alternative options, I have created only outputs and graphs based on my own initiative. 
Please inform me if any changes need to be made.

HRC-123 committed:
Added Tailwind and Output Component

ravinder-chadha commented on frontend/src/Components/Output.jsx:
```diff
@@ -0,0 +1,113 @@
+
+  const handleChange = (event) => {
+      setOption(event.target.value);
+    //   if (event.target.value == 'Graphs') {
```
Is this commented code required?

HRC-123 replied:
No Sir that is just to show outputs
I will remove in my next commit.
Thank you.

ravinder-chadha commented on frontend/src/Components/Output.jsx:
```diff
@@ -0,0 +1,113 @@
+    const [output, setOutput] = useState('Not Compiled');
+    const [files, setFiles] = useState('Still Not Compiled');
+    
+    let errorText = `Warning: Each child in a list should have a unique "key" prop.
```
Will the same error text appear for all failures?

HRC-123 replied:
This is a sample error text.
The actual error will shown when the error comes from the Backend.
Same for output, Compilation Errors, Warnings, Compilation Time etc.

ravinder-chadha reviewed (commented):
Please rectify the above in the next commit, thank you!

HRC-123 committed:
Removed Comments

HRC-123 commented:
Issues Resolved : #3 #6 
@ravinder-chadha Hello sir,
I have removed the commented code which was not useful.
I have not set some of the states and instead used there default values because of no knowledge how backend sends the response.
Please inform me if any further changes need to be made.
Thank you.

ravinder-chadha commented on frontend/src/Components/Output.jsx:
```diff
@@ -0,0 +1,107 @@
+          </div>
+        </div>
+
+        {option == "Output" && (
```
Use strict equality === instead of == to avoid unexpected type coercion.

HRC-123 replied:
I will resolve this issue in my next commit.
Thank you.

ravinder-chadha commented on frontend/src/Components/Output.jsx:
```diff
@@ -0,0 +1,107 @@
+              <p>Execution Time : {time}</p>
+            </div>
+            <div className="">
+              {error && (
```
Please use ternary operator for cleaner conditional rendering

HRC-123 replied:
I will resolve this issue in my next commit.
Thank you.

HRC-123 committed:
Converted into strict inequality,ternary operator and cleared issue of textoverflow

HRC-123 commented:
Fixes Issues  : 
- #3 
- #6 

@ravinder-chadha Hello sir,
- I have converted the code to strict equality and used ternary operator.
- I have also added code which handles the text-overflow in case of error or output.
Please inform me if any further changes need to be made.
Thank you.

ravinder-chadha reviewed (approved):
Looks fine to me!
